### DISCUSSION TITLE
Model Labels

### DISCUSSION THREAD

**Discussion by Xavitorradas** [EdgeTX/edgetx#2942] [Ideas]
Coukd be possible to add a AND/OR in Labes? If you ask for planes and helicopters and push over OR will be able to select both categories but if you push over AND you will select the filters labels you put in the models.
  AND / OR could be set on the top bars beside NEW MODELS .

**Comment by Astrohog** [comment]
Switching labels is a little frustrating for me as it can take a few seconds for the radio to respond.  I first select the label I want then wait till the radio is ready then deselect the previous label or labels.  Just as I can load a model by double tapping, it would be nice to select a label (and deselect all other active labels) by double tapping the label.
